frontend/flashfolio/src/Viewer.js

```diff
@@ -91,6 +99,8 @@ export default function Viewer({viewMode="view"}) {
 				)}`}
 			download="myDeck.json"
 		>Download</a>
+		<button onClick={homeButton}>Home</button>
+		<button onClick={loadButton}>Load Deck</button>
```

Comment:
Note: for the sake of brevity we should component-ize these items later perhaps. However no need to do that right now.